src/styles.css

```diff
@@ -1,4 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
 body {
-    background-color: lightgray;
+    background-image: url('./images/wallpaper-wood-hd-22.jpg');
+    background-repeat:no-repeat; /*we want to have one single image not a repeated one*/
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add one whitespace between the colon and the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ec2-54-187-222-128.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=css%3Aformatting)